cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/Node.kt

```diff
@@ -251,9 +249,10 @@ open class Node : IVisitable<Node>, Persistable, LanguageProvider, ScopeProvider
 
     fun addNextDFG(
         next: Node,
-        properties: MutableMap<Properties, Any?> = EnumMap(Properties::class.java)
+        granularity: GranularityType = GranularityType.FULL,
+        memberField: FieldDeclaration? = null,
```

Comment:
Here I think we need some doc. The function was simple previously but now we do have more parameters, especially what a `memberField` is an when it should be set would be good here.

Reply:
Agree. The membeField naming is a leftover. It should also be called partialTarget here.